### DISCUSSION TITLE
esptool.py version used in esp-idf 4.4.6 and arduino-esp32 core 2.0.14

### DISCUSSION THREAD

**Discussion by sboicu** [espressif/arduino-esp32#9157] [Q&A]
Hello.

Do you know why 
esp-idf 4.4.6-dirty uses version esptool.py v3.3.4-dev
and 
arduino-esp32 core in Arduino IDE uses esptool.py v4.5.1?

I try to match a generation of the bootloader from esp-idf to arduino-ide and there are small differences in bootloader based on sdk and I wonder if maybe this difference in esptool might have something to do with this.

Thanks in advance.

**Comment by P-R-O-C-H-Y** [comment]
Hello @sboicu 

I have checked out esp-idf release/v4.4 and compared to arduino release/2.x branch:
IDF uses esptool.py v4.6.2
Arduino uses esptool.py v4.5.1
As the Arduino 2.x branch was not updated for some time, I see this is all fine :)

Where did you saw that esp-idf 4.4.6-dirty uses version esptool.py v3.3.4-dev?
